### PR TITLE
Route TX history & login signature message bugfixes

### PR DESCRIPTION
**Pull request by Krysh90**

### CODE REVIEW

File: app/screens/LoginScreen.tsx [hunk 1]
```diff
@@ -208,6 +218,7 @@ const LoginScreen = () => {
                   name="userName"
                   label={t("model.user.legacy_address")}
                   returnKeyType="next"
+                  valueHook={addressValueHook}
```
Comment (ThaBrad) [inline app/screens/LoginScreen.tsx:221]: Why not simply assign it to the onChange / onChangeText param?

Comment (ThaBrad) [reply app/screens/LoginScreen.tsx:221]: also name is _a bit_ misleading (in a react project) 😝

Comment (Krysh90) [reply app/screens/LoginScreen.tsx:221]: Because it is not available on our `Input`, but `valueHook` was

Comment (ThaBrad) [reply app/screens/LoginScreen.tsx:221]: Ah, not extended actually - got it
Ok, a bit weird implementation/abstraction 😄

**Review by ThaBrad — APPROVED** [review]

**Pull request merged**